src/main/java/com/superzanti/serversync/GUIJavaFX/PaneSideBar.java

```diff
@@ -0,0 +1,74 @@
+package com.superzanti.serversync.GUIJavaFX;
+
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.scene.control.Button;
+import javafx.scene.layout.VBox;
+import javafx.scene.shape.SVGPath;
+
+// GUI of the left panel to navigate in the application
+public class PaneSideBar extends VBox {
+    // SVG for each icons of the left panel
+    String svgPathSync = "M12 4V1L8 5l4 4V6c3.31 0 6 2.69 6 6 0 1.01-.25 1.97-.7 2.8l1.46 1.46C19.54 15.03 20 13.57 20 12c0-4.42-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6 0-1.01.25-1.97.7-2.8L5.24 7.74C4.46 8.97 4 10.43 4 12c0 4.42 3.58 8 8 8v3l4-4-4-4v3z";
```

Comment:
Are these custom made SVG assets or are they copied from somewhere?

Reply:
https://material.io/resources/icons/?style=baseline

- Sync : https://material.io/resources/icons/?icon=sync&style=baseline
- Options : https://material.io/resources/icons/?icon=build&style=baseline
- Log : https://material.io/resources/icons/?icon=desktop_windows&style=baseline

Reply:
Goodo, pretty sure material is a compatible license, will have to check though.